model/distro/distro.go

```diff
@@ -593,3 +595,46 @@ func (d *Distro) GetResolvedPlannerSettings(s *evergreen.Settings) (PlannerSetti
 	d.PlannerSettings = resolved
 	return resolved, nil
 }
+
+func (d *Distro) Add(creator *user.DBUser) error {
+	err := d.Insert()
+	if err != nil {
+		return errors.Wrap(err, "Error inserting distro")
+	}
+	return d.AddPermissions(creator)
```

Comment:
wrap/withstack this

Reply:
withstack

Reply:
Could you elaborate on the advantage of having it here? I don't really see a reason to do it here since the wrapped error is enough to debug